exchanges/poloniex/poloniex_types.go

```diff
@@ -446,75 +125,906 @@ var WithdrawalFees = map[currency.Code]float64{
-	Status        string        `json:"status"`
+// CancelOrdersRequest represents cancel order request parameters
+type CancelOrdersRequest struct {
+	Symbol         currency.Pair `json:"symbol"`
+	OrderIDs       []string      `json:"ordIds,omitempty"`
+	ClientOrderIDs []string      `json:"clOrdIds,omitempty"`
 }
 
-// WalletWithdrawals holds wallet withdrawal info
-type WalletWithdrawals struct {
-	WithdrawalRequestsID int64         `json:"withdrawalRequestsId"`
-	Currency             currency.Code `json:"currency"`
-	Address              string        `json:"address"`
-	Amount               float64       `json:"amount,string"`
-	Fee                  float64       `json:"fee,string"`
-	Timestamp            types.Time    `json:"timestamp"`
-	Status               string        `json:"status"`
-	TransactionID        string        `json:"txid"`
-	IPAddress            string        `json:"ipAddress"`
-	PaymentID            string        `json:"paymentID"`
-}
-
-// TimeStampResponse returns the time
-type TimeStampResponse struct {
-	ServerTime types.Time `json:"serverTime"`
+// KillSwitchStatus represents a kill switch response
+type KillSwitchStatus struct {
+	StartTime        types.Time `json:"startTime"`
+	CancellationTime types.Time `json:"cancellationTime"`
+}
+
+// SmartOrderRequestRequest represents a smart trade order parameters
+type SmartOrderRequestRequest struct {
+	Symbol         currency.Pair `json:"symbol"`
+	Side           order.Side    `json:"side"`
+	TimeInForce    timeInForce   `json:"timeInForce,omitempty"`
+	AccountType    accountType   `json:"accountType,omitempty"`
+	Type           string        `json:"type,omitempty"`
+	Price          float64       `json:"price,omitempty,string"`
+	StopPrice      float64       `json:"stopPrice,omitempty,string"`
+	Quantity       float64       `json:"quantity,omitempty,string"`
+	Amount         float64       `json:"amount,omitempty,string"`
+	ClientOrderID  string        `json:"clientOrderId,omitempty"`
+	TrailingOffset string        `json:"trailingOffset,omitempty"` // trailing stop price offset (Suffix with%, to trailing the proportion, without%, to trailing the price distance)
+	LimitOffset    string        `json:"limitOffset,omitempty"`    // When the order is triggered, the order is issued with a limit order based on the offset of the market price. ( Suffix with%, to limit the proportion, without%, to limit the price distance)
+	Operator       string        `json:"operator,omitempty"`       // Activation the price operator when orderType is TRAILING_STOP or TRAILING_STOP_LIMIT. Possible values are: GTE - Greater than or equal and LTE - Less than or equal
+}
+
+// CancelReplaceSmartOrderRequest represents a cancellation and order replacement request parameter for smart orders.
+type CancelReplaceSmartOrderRequest struct {
+	orderID          string  `json:"-"` // orderID: will be used in request path.
+	OldClientOrderID string  `json:"-"` // client order ID of the order to be replaced.
```

Comment:
🚧 Change request:
* It's not _old_ yet
* Not sure if we need a comment if we name it better
```suggestion
	ReplaceClientOrderID string  `json:"-"
```